www/src/components/copy.js

```diff
@@ -13,15 +13,21 @@ import {
 } from "../utils/presets"
 
 const copyToClipboard = content => {
```

Comment:
NOT a blocking comment!

no big deal for now but have you seen the async clipboard api?
https://googlechrome.github.io/samples/async-clipboard/
https://alligator.io/js/async-clipboard-api/

It has pretty good support and has less bugs than the old api:
https://www.chromestatus.com/feature/5861289330999296

We can feature detect this and if browsers don't support it we don't show it. I don't think we should support old clipboard api.

Reply:
Nice. I was thinking of extracting this into a utility, and this seems like the kind of "branching logic" that can be tested and extracted. I'll do this in a bit -- love this idea!

Reply:
Done in #16012